protos/sources/vega/markets.proto

```diff
@@ -438,4 +438,8 @@ message CompositePriceConfiguration {
   repeated string source_staleness_tolerance = 5;
   // Which method is used for the calculation of the composite price for the market.
   CompositePriceType composite_price_type = 6;
+  // Additional price sources to be used for index price calculation.
+  repeated vega.DataSourceDefinition data_sources_spec = 7;
+  // For each price source the corresponding binding 
```

Comment:
```suggestion
  // List of each price source and its corresponding binding
```

Reply:
Is this a good way to describe it?

Reply:
you tell me :) for each price source in data_sources_spec, the binding at the corresponding index defines the binding (i.e. the property name defining the price)